rust/macros/module.rs

```diff
@@ -213,6 +213,30 @@ fn permissions_are_readonly(perms: &str) -> bool {
     }
 }
 
+#[test]
```

Comment:
Usually tests would be a in `#[cfg(test)] mod tests {}` is there a reason not to?

Reply:
Yeah, I was not sure what to do, since these did not need extra items defined. But I guess I can see being consistent being better (as well putting them at the bottom of the file).